cellbender/remove_background/train.py

```diff
@@ -185,29 +210,15 @@ def run_inference(dataset_obj: SingleCellRNACountsDataset,
                         output_dim=args.z_dim,
                         input_transform='normalize')
 
-    encoder_d = EncodeD(input_dim=count_matrix.shape[1],
-                        hidden_dims=args.d_hidden_dims,
-                        output_dim=1,
-                        log_count_crossover=
-                        dataset_obj.priors['log_counts_crossover'])
-
-    if args.model == "simple":
-
-        # If using the simple model, there is no need for p.
-        encoder = CompositeEncoder({'z': encoder_z, 'd_loc': encoder_d})
-
-    else:
+    encoder_other = EncodeNonZLatents(n_genes=count_matrix.shape[1],
+                                      z_dim=args.z_dim,
+                                      hidden_dims=[100, 50],
```

Comment:
magic number => consts (or expose to CLI)

Reply:
Will put it in consts, as it really shouldn't need to be modified by users